src/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Children } from 'react'
+
+export default function Button({Children,className,disabled}) {
```

Comment:
we can also add the styles for button like height , width and size .

Reply:
yes i am on it now, it was just a demo right now